### PR TITLE
formpalyer docker revamp

### PR DESCRIPTION
**Pull request by snopoke**
## Summary
Most of the changes here only impact the docker-compose files or docker localsettings. The only HQ change is to allow specifying and separate formplayer URL for web apps since web apps communicates with formplayer from the browser which is outside of the docker network.

Formplayer PR: https://github.com/dimagi/formplayer/pull/909

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Review by kaapstorm — APPROVED** [review]
<3

File: docker/hq-compose.yml [hunk 1]
```diff
@@ -100,14 +104,15 @@ services:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.13/data
 
   kafka:
-      image: wurstmeister/kafka:0.8.2.2-1
+      image: wurstmeister/kafka:2.13-2.6.0
```
Comment (millerdev) [inline docker/hq-compose.yml:107]: Is this related to the formplayer revamp?

**Review by millerdev — APPROVED** [review]

File: docker/hq-compose.yml (hunk 1 above)
Comment (snopoke) [reply docker/hq-compose.yml:107]: It was not - I did initially have some trouble getting kafka to work with commcare running in docker but I think our config was just wrong. I can revert this change though this is now the same version we're using on prod.

Comment (millerdev) [reply docker/hq-compose.yml:107]: Fine to keep it, assuming I'm understanding correctly that it's basically cleanup.
